### ISSUE TITLE
Warning thrown when using self-signed certificates

### ISSUE THREAD

**Issue by emzet** [SonarSource/sonar-scanner-cli-docker#140]
There is the [guide](https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/) about using self-signed certificates. When one runs provided command:

````
docker run \
  --rm \
  -v ${YOUR_CERTS_DIR}/cacerts:/tmp/cacerts \
  -v ${YOUR_REPO}:/usr/src \
  -e SONAR_HOST_URL="http://${SONARQUBE_URL}" \
  sonarsource/sonar-scanner-cli
````
got this warning message:
```
Warning: use -cacerts option to access cacerts keystore
```
Besides that, everything works as it should.

**Comment by carminevassallo** [comment]
Dear @emzet,

Thanks for reporting this issue! As you might have noticed in the `README.md`, we are trying to centralize the management of all bug reports and feature requests related to Sonar products in our [forum](https://community.sonarsource.com/).

If you think this issue is still relevant, could you please post it there? In the meanwhile, I'm closing it.
